### DISCUSSION TITLE
Can webpack watch for changes in workspace dependencies?

### DISCUSSION THREAD

**Discussion by dgnemo** [nrwl/nx#12218] [General]
I am trying to separate a large project into multiple packages within a workspace.
I am using the Package-Based approach (with PNPM).

One issue I am facing is that webpack is not detecting changes in the package dependencies, thus it's not automatically recompiling the main app during development.

Is there any solution to this?
Thanks

**Comment by ffMathy** [comment]
I'd like to know this too.
